<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>776</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1111111</width>
    <height>1111111</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>421</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
border-radius:px;
}
QWidget#login

 {
	background-color: rgb(255, 255, 255);
 }
QWidget#regedit

 {
	background-color: rgb(255, 255, 255);
 }
QTabBar::tab {
    min-width: 210px; /* 选项卡的最小宽度 */
    min-height: 30px; /* 选项卡的最小高度 */
}
QTabBar::tab:selected {
    background-color: rgb(233, 255, 247); /* 选中选项卡的背景色 */
    color: black; /* 选中选项卡的文本颜色 */
}
QTabBar::tab
{
    background-color:white; /* 选中选项卡的背景色 */
    color: black; /* 选中选项卡的文本颜色 */
}


</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="iconSize">
    <size>
     <width>39</width>
     <height>39</height>
    </size>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideLeft</enum>
   </property>
   <property name="usesScrollButtons">
    <bool>false</bool>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <property name="tabsClosable">
    <bool>false</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="login">
    <property name="styleSheet">
     <string notr="true">
QComboBox,QLineEdit
{
	border:none;
	border-radius:20px;
	
	background-color: rgb(233, 255, 247);
}
QPushButton
{
	border:none;
	border-radius:20px;
	
	background-color: rgb(4, 212, 167);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <attribute name="title">
     <string>登录</string>
    </attribute>
    <widget class="QPushButton" name="btnlogin">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>320</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>152</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>1111111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>  请输入账号/手机号码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>236</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>1111111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>  请输入密码</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="regedit">
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	border:none;
	border-radius:20px;
	background-color: rgb(233, 255, 247);
}
QPushButton
{
	border:none;
	border-radius:20px;
	
	background-color: rgb(4, 212, 167);
	color: rgb(255, 255, 255);
}
</string>
    </property>
    <attribute name="title">
     <string>注册</string>
    </attribute>
    <widget class="QLineEdit" name="password_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>210</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>11111111</width>
       <height>11111111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>  请输入密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_3">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>141</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>1111111</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="placeholderText">
      <string>  请输入手机号码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnreg">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>320</y>
       <width>350</width>
       <height>40</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>431</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background-color: rgb(55,100, 0);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/sourse_images/2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>60</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
border:none;
background-color: transparent;
}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;欢迎登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
